linode/manager
# change: [M3-7688] - Gravatar Analytics

## Description 📝
- Adds analytics to Gravatars 👤 
  - Adds one event that says whether or not the current user has a Gravatar when the page loads
  - Adds one event for when the users clicks `Manage photo` on the Profile Display page (`http://localhost:3000/profile/display`)

## How to test 🧪

### Prerequisites
- Run `window._satellite.setDebug(true)` in your browser console
- Refresh your page

### Verification steps

| Before  | After   |
| ------- | ------- |
| When the page loads, verify you see an event like this if you have a Gravatar | ![Screenshot 2024-04-18 at 3 57 07 PM](https://github.com/linode/manager/assets/115251059/bf1ab9e9-b062-445c-9a3f-27c9e73ee8a4) |
| When the page loads, verify you see an event like this if you *don't* have a Gravatar | ![Screenshot 2024-04-18 at 4 01 36 PM](https://github.com/linode/manager/assets/115251059/092477e7-abf9-4bc0-9d72-5cd1ede7c9c6) |
| Verify you see this event when you click `Manage Photo` on the `http://localhost:3000/profile/display` page | ![Screenshot 2024-04-18 at 4 04 14 PM](https://github.com/linode/manager/assets/115251059/515a2a3c-72a9-4cd0-9a68-41e529ea0372) |



## As an Author I have considered 🤔
